chapters/methodology.tex

```diff
@@ -14,3 +14,25 @@ \subsection{State Machine}
 \subsection{User Interface}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
+\subsection{Model Training}
+
+As stated in the background section, soHappy uses a deep learning model to 
+provide information about a person smiling by utilizing TensorFlow.
+
+However, for TFLite to detect smiling, the model must be trained first. This
+model is not trained on the device, but will be trained in before and shipped
+with the android app.
+
+The soHappy smile detection model is based a convolutional neural network 
+(CNN). CNNs are useful for image analysis purpuses. A CNN utilizes multiple
+convolutional layers as well as pooling layers. A convolutional layer works by
+looking at regions of an image instead of individual pixels. This process is
+called convolution. A more in depth explanation on how CNN work, see 

```

Comment:
Unless I'm mistaken, I thought convolution is the process in which a filter is applied onto an image.

I'd suggest rewriting "A more in depth explanation on how CNN work, see [...]" into something like:
"Refer to `\cite{8308186}` for a more in-depth explanation on how CNNs work.

Reply:
On the first part: Convolution is actually just looking at neighbour pixels, but a convolution layer does this for applied filters. I will extent this paragraph.